panel/applets/icon-tasklist/Application.vala

```diff
@@ -0,0 +1,283 @@
+public class Application: AttentionStatusListener, IconChangeListener, ScreenProvider{
```

Comment:
Oh definitely copyright it to you! It's your hard work, after all. Gotta make sure you're credited for it :D